Mathlib/CategoryTheory/Category/Cat.lean

```diff
@@ -105,6 +107,16 @@ theorem comp_map {C D E : Cat} (F : C ⟶ D) (G : D ⟶ E) {X Y : C} (f : X ⟶
 set_option linter.uppercaseLean3 false in
 #align category_theory.Cat.comp_map CategoryTheory.Cat.comp_map
 
+@[simp]
+lemma whiskerLeft_app {C D E : Cat} (F : C ⟶ D) {G H : D ⟶ E} (η : G ⟶ H) (X : C) :
+    (F ◁ η).app X = η.app (F.obj X) :=
+  CategoryTheory.whiskerLeft_app F η X
```

Comment:
When the proof can be `rfl`, the proof should be `rfl`.
```suggestion
    (F ◁ η).app X = η.app (F.obj X) := rfl
```
Could you do the same modification for the other lemmas (above and below this one)? Could you also add similar lemmas for the associator and unitors?

Reply:
Will we really need similar lemmas for associators and unitors? Because since `Cat` is a strict bicategory there are already simp-lemmas that these are just eqToIsos.

Reply:
(although their proofs are by aesop_cat and not rfl, so maybe thats a reason to add them here)